modules/nf-core/last/mafconvert/main.nf

```diff
@@ -31,7 +36,20 @@ process LAST_MAFCONVERT {
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
     set -o pipefail
-    maf-convert $args $format $maf | gzip --no-name > ${prefix}.${format}.gz
+
+    case $format in
+        bam)
+            maf-convert $args -d sam  $maf | samtools view -b -o ${prefix}.${format}
+            ;;
+        cram)
+            # CRAM output is not supported if the genome is compressed with something else than bgzip
+            samtools faidx $fasta
```

Comment:
The fasta index should be an input file, it's not efficient to create it every time you run this process

Reply:
Good point.  I will soon push a fix.